Makefile

```diff
@@ -13,5 +13,8 @@ build:
 	@cp -Rf build examples/blog/
 	@echo "Files build/ng-admin.min.css and build/ng-admin.min.js updated (with minification)"
 
+build-dev:
+	@NODE_ENV=production ./node_modules/webpack/bin/webpack.js --progress
```

Comment:
Otherwise you can launch the watch method and embed an URL such as `localhost:8080/ng-admin.min.js` in your project webpack file.